src/plugins/data/public/types.ts

```diff
@@ -43,6 +43,7 @@ export interface DataSetupDependencies {
 
 export interface DataStartDependencies {
   uiActions: UiActionsStart;
+  usageCollection?: UsageCollectionSetup;
```

Comment:
No need to add this is a start dependency.
The setup phase of the plugin already receives `usageCollection`, so you could store it in a property and consume it in the start phase.